fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/PwaRobot.kt

```diff
@@ -4,24 +4,32 @@
 
 package org.mozilla.fenix.ui.robots
 
+import android.util.Log
 import androidx.test.uiautomator.UiSelector
 import org.junit.Assert.assertTrue
 import org.mozilla.fenix.helpers.AppAndSystemHelper.isExternalAppBrowserActivityInCurrentTask
+import org.mozilla.fenix.helpers.Constants
+import org.mozilla.fenix.helpers.Constants.TAG
 import org.mozilla.fenix.helpers.MatcherHelper.assertUIObjectExists
+import org.mozilla.fenix.helpers.MatcherHelper.itemWithResId
 import org.mozilla.fenix.helpers.TestHelper.mDevice
 import org.mozilla.fenix.helpers.TestHelper.packageName
 
 class PwaRobot {
-    fun verifyCustomTabToolbarIsNotDisplayed() = assertUIObjectExists(customTabToolbar(), exists = false)
-    fun verifyPwaActivityInCurrentTask() = assertTrue(isExternalAppBrowserActivityInCurrentTask())
+    fun verifyCustomTabToolbarIsNotDisplayed() = assertUIObjectExists(itemWithResId("$packageName:id/toolbar"), exists = false)
+    fun verifyPwaActivityInCurrentTask() {
+        Log.i(TAG, "Trying to verify that the latest activity of the application is used for custom tabs or PWAs")
+        assertTrue(isExternalAppBrowserActivityInCurrentTask())
```

Comment:
Maybe we should start adding assertion error messages as well, inside the assertTrue method. Otherwise we will not get a nice message when the test fails here.

The first log ("Trying to verify that the latest activity of the application is used for custom tabs or PWAs") can be added inside the isExternalAppBrowserActivityInCurrentTask method.

Reply:
Done !